FRBDK/Glue/Glue/CodeGeneration/StateCodeGenerator.cs

```diff
@@ -530,21 +530,19 @@ private static Dictionary<string, StateSaveCategory> GetAllStateCategoryNames(IE
 
 
 
-        private static bool IsStateDefinedInBase(IElement element, string enumName)
+        private static bool IsStateDefinedInBase(GlueElement element, string enumName)
         {
-            bool toReturn = false;
             if (string.IsNullOrEmpty(element.BaseElement))
             {
-                toReturn = false;
+                return false;
```

Comment:
No "early outs" unless explicitly at the very beginning of the method. toReturn is intentional.

Reply:
it is at the very beginning of the method. See line 533-535

Reply:
I found a coding document, cleaned it up. See the //////////early out//////// section for examples:

https://docs.flatredball.com/flatredball/contributing/general-programming-flatredball-programming-style-guide#patterns